### PR TITLE
added ability to turn off persistence. Waiting on Chris for final changes

### PR DESCRIPTION
**Pull request by cloudformdesign**
This updates the state mixin to be aware of the use_persistence variable

Per Chris's request, this variable is invisible to the user for all blocks where it is disabled. **However** I have left it visible for the StateChange block itself, as it is not disabled by default there. **Please Comment**

### CODE REVIEW

File: state_mixin.py [hunk 1]
```diff
@@ -14,6 +15,9 @@ class StateMixin(object):
     state_expr = ExpressionProperty(title='State Expression', default='{{$state}}')
     backup_interval = TimeDeltaProperty(title='Backup Interval',
                                         default={'seconds': 600})
+    load_from_persistence = BoolProperty(title="Load from persistence?", default=True,
```
Comment (mattdodge) [inline state_mixin.py]: If this is in the `StateMixin` parent class, why does it need to get added to each block too?

Comment (cloudformdesign) [inline state_mixin.py]: It probably doesn't need to. Are you interested in adding this feature now? I will put it in.

Comment (mattdodge) [inline state_mixin.py]: Since it gets added as part of this PR, it would be best to push another commit that removes it.

If it was an outstanding issue I would say leave it but since this is the pull that causes it we should take care of it

**Comment by mattdodge** [comment]
This is handled by the recent refactor to the state blocks, closing out
